virttest/qemu_monitor.py

```diff
@@ -2506,3 +2506,26 @@ def drive_backup(self, device, target, format, sync, speed=0,
         if speed:
             args["speed"] = speed
         return self.cmd(cmd, args)
+
+    def press_release_key(self, key, down=True):
```

Comment:
can you rename  function name to send_release_key ?

Reply:
I use "press", because it means "press and hold this key", it will not release until "down=False" . 
If use "send", I think it doesn't have the meaning of keep holding, seems the same function as sendkey()